rpm/linux-glibc-devel.spec

```diff
@@ -38,12 +37,10 @@ low level system software, and is only used indirectly by regular
 applications.
 
 %prep
-%setup -q -n linux-%{kversion}
-# api-fix-compatibility-of-linux-in.h-with-netinet-in.patch
-%patch0 -p1
+%autosetup -p1 -n %{name}-%{version}/upstream
 
 %build
-make allyesconfig
+make mrproper
```

Comment:
This I'm not following. Why change? Make mrproper should be doing some cleaning, but there isn't anything really build? Older version actually did some config stuff.

Reply:
I made a test build with `make allyesconfig` for i486 -- the RPM packages are file-by-file identical (save the timestamped and commit-ref'd path for `COPYING`). I also checked OpenSUSE's .spec file, and it doesn't contain any `make` commands. I suspect that change is just a development leftover.

I'd say it can be left as-is (and that way we don't have to make yet another build round).

Reply:
I restored `allyesconfig` here because `mrproper` doesn't make sense.

Reply:
You can remove the line completely. Userspace header are not affected by kernel configuration.